format-common/src/main/java/io/cdap/plugin/format/plugin/FileSinkProperties.java

```diff
@@ -72,6 +73,14 @@ default void validate(FailureCollector collector, Map<String, String> arguments)
    */
   String getPath();
```

Comment:
do we ever call this anymore? If not, we need to annotate it as deprecated and update the javadoc. If so, we need to update the javadoc and describe how it is different than the new method.

Reply:
This method is no different from the added method and is not called, with the added method you can have additional information about runtime arguments, marked it as deprecated.